week2/yeonwook/index.css

```
@@ -0,0 +1,152 @@
+button{
+    background: none;
+    border: none;
+    padding: 0;
+    cursor: pointer;
+    outline: inherit;
+}
+
+.uppersection{
+    position: relative;
+}
+
+.upperheader{
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    padding: 12px 16px;
+    box-sizing: border-box;
```

box-sizing: border-box는 일일이 요소에 다 설정하는것보다 

```suggestion
    * {
        box-sizing: border-box;
     }
```
전체 요소에 적용 시켜주고 시작하는 것이 편합니다!